seafile7+.json

```
@@ -0,0 +1,61 @@
+{
+  "Seafile v7+": {
+    "containers": {
+      "seafile-mariadb": {
+        "image": "mariadb",
+        "tag": "10.1",
+        "launch_order": 1,
+        "volumes": {
+          "/var/lib/mysql": {
+            "description": "Share for MariaDB data persistent store.",
+            "label": "MariaDB Storage [seafile-db]"
+          }
+        },
+        "opts": [
+          ["-e", "MYSQL_LOG_CONSOLE=true"],
+          ["-e", "MYSQL_ROOT_PASSWORD=MDA0NDRmNjYxOTMyMzJiOTY5MzZkNGY4"]
+        ]
+      },
+      "seafile-memcached": {
+        "image": "memcached",
+        "tag": "1.5.6",
+        "launch_order": 2,
+        "cmd_arguments": [["memcached", "-m 256"]]
+      },
+      "seafile-mc": {
+        "image": "seafileltd/seafile-mc",
+        "launch_order": 3,
+        "ports": {
+          "80": {
+            "description": "Seafile WebUI port. Suggested default: 8080",
+            "host_default": 8080,
+            "protocol": "tcp",
+            "label": "WebUI port",
+            "ui": true
+          }
+        },
+        "volumes": {
+          "/shared": {
+            "description": "Share for Seafile for persistent data.",
+            "label": "Seafile Storage [seafile-data]"
+          }
+        },
+        "opts": [
+          ["-e", "DB_HOST=mariadb"],
+          ["-e", "DB_ROOT_PASSWD=MDA0NDRmNjYxOTMyMzJiOTY5MzZkNGY4"],
+          ["-e", "SEAFILE_SERVER_LETSENCRYPT=false"],
+          ["--link", "seafile-mariadb:mariadb"],
+          ["--link", "seafile-memcached:memcached"]
+        ]
+      }
+    },
+    "version": "latest",
+    "description": "Secure file sharing and hosting. <p>Based on the official <a href='https://hub.docker.com/r/seafileltd/seafile-mc' target='_blank'>Seafile multi-container image</a> for versions 7 and above. Use MariaDB and Memcached as separate container.</p> <p>HTTP only, to add HTTPS, you can use Nginx-proxy-manager.</p>",
```

Add supported architecture:
```suggestion
    "description": "Secure file sharing and hosting. <p>Based on the official <a href='https://hub.docker.com/r/seafileltd/seafile-mc' target='_blank'>Seafile multi-container image</a> for versions 7 and above, available for amd64/x86_64 architecture only. Use MariaDB and Memcached as separate container.</p> <p>HTTP only, to add HTTPS, you can use Nginx-proxy-manager.</p>",
```